src/main/java/sirius/biz/datapipes/memory/Allocator.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.datapipes.memory;
+
+import sirius.kernel.nls.NLS;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+public class Allocator {
```

Comment:
javadoc: what is the purpose of this class ? 
javadoc for all new classes ?! and and some of the methods ?